### ISSUE TITLE
Update twitter's icon

### ISSUE THREAD

**Issue by akshay-99h** [carmelopullara/react-feather#109]
I was working on a project that uses react-feather. It still shows the bird logo. Now that the logo of twitter has changed to "X", I think we can update it to X here as well.

**Comment by milinddhamu** [comment]
src/icons/twitter.js , file directory for reference

**Comment by imMatheus** [comment]
I don't think it should be changed, the thing is that the "Twitter" logo is still the bird, the platform changed its name to "X" so if you want the new icon it should not be that the Twitter component changes as you very well might want the bird logo. But instead, a new icon should be added for the X logo. The question is what to name it, X is already taken, so if it were to be added maybe something like <TwitterX /> or something like that.

**Comment by milinddhamu** [comment]
i updated path to new logo in main file i mentioned above , they are not reviewing it. I opened a PR already.

**Comment by imMatheus** [comment]
There is a Issue for it already on the main feather repo. I would guess that they are going to add it once it has been added to the main Feather repo first [https://github.com/feathericons/feather/issues/1207](https://github.com/feathericons/feather/issues/1207)
